src/lib/OpenEXR/ImfMisc.cpp

```diff
@@ -131,7 +131,7 @@ bytesPerDeepLineTable (const Header &header,
     {
         const int ySampling = abs(c.channel().ySampling);
         const int xSampling = abs(c.channel().xSampling);
-        const int pixelSize = pixelTypeSize (c.channel().type);
+        uint64_t pixelSize = pixelTypeSize (c.channel().type);
```

Comment:
Did you remove the "const" for a reason?

Reply:
Oops. It's back again